testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/adapter/servlet/OIDCPublicKeyRotationAdapterTest.java

```diff
@@ -190,6 +195,10 @@ public void testClientWithJwksUri() throws Exception {
     // KEYCLOAK-3824: Test for public-key-cache-ttl
     @Test
     public void testPublicKeyCacheTtl() {
+        driver.manage().timeouts().pageLoadTimeout(1000, TimeUnit.SECONDS);
```

Comment:
Do we need to set this timeout? I guess this is being set for debugging purposes?

Reply:
As far as i remember (the PR was written a while ago), i was banging my head against the wall because of this :P 
If i remember correctly, it was passing the test locally, but it was failing while run by the github actions (however not in all modes) for a reason i couldn't understand at that time. I'll remove it, and see if things have changed and this doesn;t fail anymore.

Reply:
It seems to be ok without this line! I set this conversation as "resolved"